toolkits/global/packages/global-autocomplete/js/index.js

```diff
@@ -120,8 +123,11 @@ const autoComplete = arguments_ => {
 
 	const generateSuggestions = data => {
 		removeSuggestions();
+		data = toSuggestionsCallback(data);
```

Comment:
I would suggest this be optional, as the server might already be returning an array, or the consumer might be using static data